Judge.gd

```diff
@@ -32,14 +32,18 @@ func score_input(input):
 		return
 	
 	last_beat_for_input_received = closest_beat
-	print("Scoring beat ", closest_beat)
-	print("Last beat", last_beat_for_input_received)
+	
+	var target_keycode = current_goal.key_sequence[closest_beat - 1]
+	
+	if not target_keycode:
+		return Score.NA
+	elif input.keycode != target_keycode:
+		return Score.MISS
```

Comment:
If we don't have a target for this beat, show nothing. If you hit the wrong key, that's a miss.